builder/vsphere/common/step_shutdown.go

```diff
@@ -16,16 +16,18 @@ import (
 )
 
 type ShutdownConfig struct {
-	// Specify a VM guest shutdown command. VMware guest tools are used by
-	// default.
+	// Specify a VM guest shutdown command. This command will be executed using
+	// the `communicator`. Otherwise the VMware guest tools are used to gracefully
+	// shutdown the VM guest.
 	Command string `mapstructure:"shutdown_command"`
 	// Amount of time to wait for graceful VM shutdown.
 	// Defaults to 5m or five minutes.
+	// This will likely need to be modified if the `communicator` is 'none'.
 	Timeout time.Duration `mapstructure:"shutdown_timeout"`
 	// Packer normally halts the virtual machine after all provisioners have
 	// run when no `shutdown_command` is defined. If this is set to `true`, Packer
 	// *will not* halt the virtual machine but will assume that you will send the stop
-	// signal yourself through the preseed.cfg or your final provisioner.
+	// signal yourself through a script or the final provisioner.
```

Comment:
```suggestion
	// signal yourself through a preseed.cfg, a script or the final provisioner.
```
What do you think of keeping the pressed.cfg as one of the options?

Reply:
agreed